src/components/SpinnerIfStandardSelectorsNotLoaded.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import SpinnerIfNotLoaded from '../components/SpinnerIfNotLoaded';
+import { getSelectedOrganisation} from '../reducers';
+import { useSelector } from "react-redux";
+
+interface Props {
+    loaded: boolean;
```

Comment:
I think we can also argue that this `loaded` parameter here is not necessaary for this `SpinnerIfStandardSelectorsNotLoaded` component as this component would only check for some default/standard settings.